packages/Notification/Notification.md

```diff
@@ -17,6 +23,8 @@ By default, notifications will be displayed in the `instructional` variant.
   not use other complex components or paragraphs.
 - Use complete sentences, while keeping the content succinct.
```

Comment:
"When there are multiple notifications, the messages should be served top to bottom from most to least important. Remove any unnecessary information such as promotional messages that add to the noise"

Should be added after this line